app/views/admin/trends/statuses/_status.html.haml

```diff
@@ -22,6 +22,9 @@
     - if status.language.present?
       •
       = standard_locale_name(status.language)
+    - if status.trendable? && !status.account.discoverable?
+      •
+      = t('admin.trends.statuses.not_discoverable')
```

Comment:
When would that be shown exactly?

Reply:
Users can opt-in and out of discoverability